test/ServiceBus.Tests/EvictionStrategyTests/EHPurgeLogicTests.cs

```diff
@@ -233,7 +230,15 @@ private Task AddDataIntoCache(EventHubQueueCacheForTesting cache, int count)
                 count--;
```

Comment:
HE sequence numbers are ascending.  Why are they descending in this test?

Reply:
the tests doesn't care about sequenceToken and such. I added those to fill in the fields so that runtime won't throw NullReferenceException when try to serialize eventData. Those fields are automatically filled by calling `new EventData()`in in full dotnet, but not anymore in netstandard. Hence those new lines

Reply:
Sequence numbers do effect cache behavior, so having them in reverse order in tests seems dangerous.  Basically we're generating test data that violates the expectations of the cache.  The fact that it works in this case is, imo, not a good reason to test against invalid data.

Reply:
I will fix this